app/views/feeds/_form_audio_format.html.erb

```diff
@@ -9,7 +9,9 @@
       <div class="row">
         <div class="col-3">
           <div class="form-floating input-group" aria-label="audio type">
-            <%= form.select :audio_type, audio_format_options, {include_blank: true}, disabled: local_assigns[:lock_format], data: {action: "toggle-field#displayOnlyThisField"} %>
+            <% html_options = {data: {action: "toggle-field#displayOnlyThisField"}} %>
+            <% html_options[:disabled] = true if local_assigns[:lock_format] %>
+            <%= form.select :audio_type, audio_format_options, {include_blank: true}, html_options %>
```

Comment:
if disabled is set to false, then even if it should be disabled (e.g. locked), then it shows is enabled.
Instead only set a disabled option when setting it to true.

Reply:
Ah, yes the formbuilder respecting any val you pass as an override.  Good catch!